# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../licq/compile-fix.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur licq-1.3.2-orig/plugins/email/src/forwarder.cpp licq-1.3.2/plugins/email/src/forwarder.cpp
--- licq-1.3.2-orig/plugins/email/src/forwarder.cpp	2005-10-20 21:57:43.000000000 +0000
+++ licq-1.3.2/plugins/email/src/forwarder.cpp	2006-01-19 16:32:17.606821750 +0000
@@ -373,7 +373,7 @@
     if (nPPID == LICQ_PPID)
       sprintf (szFrom, "From: \"%s\" <%s@pager.icq.com>", u->GetAlias(), u->IdString());
     else
-      sprintf (szFrom, "From: \"%s\" <%s>", u->GeAlias(), u->GetEmailPrimary());
+      sprintf (szFrom, "From: \"%s\" <%s>", u->GetAlias(), u->GetEmailPrimary());
     sprintf (szReplyTo, "Reply-To: \"%s %s\" <%s>", u->GetFirstName(), u->GetLastName(), u->GetEmailPrimary());
   }
   sprintf (szDate, "Date: %s", ctime(&t));
